tests/S3/CustomCachingStream.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Aws\Test\S3;
+
+use GuzzleHttp\Psr7\Stream;
+
+/**
+ * Class CustomCachingStream
+ *
+ * @package S3
```

Comment:
The package annotation shouldn't be used. It's only there to replace namespaces in php 5.2, and you've missmatched it with the namespace here anyway.